netbox/ipam/forms/filtersets.py

```diff
@@ -224,12 +224,13 @@ class PrefixFilterForm(TenancyFilterForm, NetBoxModelFilterSetForm):
     site_id = DynamicModelMultipleChoiceField(
         queryset=Site.objects.all(),
         required=False,
-        null_option='None',
-        query_params={
-            'region_id': '$region_id'
-        },
         label=_('Site')
     )
+    location_id = DynamicModelMultipleChoiceField(
```

Comment:
wondering if these could be a mixin ScopedFilterSet as we are going to be using this in several models and they will all have the same filter, form fields etc.. could have site_id, location_id, site_group all in the mixin.

Reply:
It's not yet clear which models will support this same set of scope types. VLANGroup, for instance, supports these four but also racks, cluster groups, and clusters. We'll utilize mixins at some point for sure but it's probably still too early for that degree of optimization.